crates/bindings/src/lib.rs

```diff
@@ -2,10 +2,16 @@ use alloy_sol_types::sol;
 
 sol!(
     #![sol(all_derives = true)]
+    #[derive(serde::Serialize, serde::Deserialize)]
     IOrderBookV3, "../../out/IOrderBookV3.sol/IOrderBookV3.json"
 );
 
 sol!(
     #![sol(all_derives = true)]
     IERC20, "../../out/IERC20.sol/IERC20.json"
 );
+
+sol!(
+    #![sol(all_derives = true)]
+    ERC20, "../../out/ERC20.sol/ERC20.json"
```

Comment:
shouldn't need this -- we have IERC20

Reply:
we need it, because we need to get the decimals of tokens specified in a `Deployment` struct when converting it to `IO`

Reply:
right gotcha